integration-test/step_definitions/support/iuv_steps.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('assert')
+const {Given, When, Then} = require('@cucumber/cucumber')
+const {generateIUV} = require("./iuv_client");
+const {randomOrg} = require("./common");
+
+let responseToCheck;
+let iuv;
+let idOrg;
+
+// Given
+Given('IUV Generator Function running', async function () {
+	//TODO health check call
```

Comment:
[use this endpoint](https://github.com/pagopa/pagopa-iuv-generator/blob/main/src/main/java/it/gov/pagopa/iuvgenerator/Info.java#L24)